run_cupy_install_test.py

```diff
@@ -17,7 +17,7 @@
     'cudnn': docker.cudnn_choices + ['cudnn-latest-with-dummy'],
     'nccl': docker.nccl_choices,
     'numpy': ['1.9', '1.10', '1.11', '1.12'],
-    'cython': [None, '0.21', '0.24', '0.25'],
```

Comment:
Do you know why 0.21 is kept, while 0.22 and 0.23 are skipped? Depending on the reason, we should not remove 0.21.